src/content/docs/en/reference/modules/astro-actions.mdx

```diff
@@ -301,3 +301,27 @@ export const onRequest = defineMiddleware(async (context, next) => {
 </p>
 
 `deserializeActionResult()` will reverse the effect of `serializeActionResult()` and return an action result to its original state. This is useful to access the `data` and `error` objects on a serialized action result.
+
+### `getActionPath()`
+
+<p>
+
+**Type:** `(action: ActionClient<any, any, any>) => string`
+<Since v="5.1.0" />
+</p>
+
+The `getActionPath()` utility is used to get the path to an action, taking base and trailing slash settings into account. This is useful when you want to call an action for more advanced usages, such as providing custom headers or not using the fetch API (eg. `navigator.sendBeacon`):
+
+```ts
+import { actions, getActionPath } from 'astro:actions'
+
+const path = getActionPath(actions.like) // '/_actions/like'
```

Comment:
I'm not sure what the commented part is representing. Is that what is returned by `getActionPath(actions.like)`?

How is the example different using `getActionPath(actions.like)` vs just `actions.like`. You mention that often you can just use the action. What is happening here that we're benefiting from using `getActionPath()` or that wouldn't otherwise be possible?

Reply:
> I'm not sure what the commented part is representing. Is that what is returned by getActionPath(actions.like)?

Yes!

> How is the example different using getActionPath(actions.like) vs just actions.like. You mention that often you can just use the action. What is happening here that we're benefiting from using getActionPath() or that wouldn't otherwise be possible?

See the `keepalive` option. I guess it's not clear enough tho, so i'll probably use an example with `sendBeacon` instead

Reply:
Ha I see (I think)!
> How is this example different using getActionPath than not using it?
Calling an action directly does the following under the hood:
```js
await fetch(someInternalPathLogic, {
  method: 'POST',
  headers: {
    'Content-Type': 'application/json',
  },
})
```
This is internal, which means you can't pass custom headers (eg. you want to protect your actions with a auth header) nor custom options (eg. keepalive which prevents the request from being killed).

Does that help?

Reply:
That is very helpful, thank you!